tests/unit_tests/test_ecephys.py

```diff
@@ -215,3 +216,24 @@ def test_check_spike_times_not_in_unobserved_interval_multiple_units():
         object_name="TestUnits",
         location="/",
     )
+
+
+def test_check_ascending_spike_times_pass():
+    units_table = Units()
+    units_table.add_unit(spike_times=[0.0, 0.1])
+    units_table.add_unit(spike_times=[1.0, 2.0])
+    assert check_ascending_spike_times(units_table=units_table) is None
+
+
+def test_check_ascending_spike_times_fail():
```

Comment:
These checks look good - for coverage, can you add one more that the missing spike times is properly skipped (i.e., a simple test with a Units table with no units, checking that the function returns None and does not error out)

Reply:
Sure! Thank you for the suggestion